src/i18n/pt-br/nav.ts

```diff
@@ -1,75 +1,78 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	startHere: 'Comece Aqui',
+	welcomeWorld: 'Bem-vindo, Mundo!',
 	'getting-started': 'Introdução',
-	install: 'Instalação',
-	'editor-setup': 'Configuração do Editor',
-	'upgrade-astro': 'Atualize o Astro',
+	'install-and-setup': 'Instalação e Configuração',
+	'guides/deploy': 'Faça deploy do seu site',
+	'guides/migrate-to-astro': 'Migre para o Astro',
+	'blog-tutorial': 'Tutorial: Construa um Blog',
 
 	coreConcepts: 'Principais Conceitos',
 	'concepts/why-astro': 'Por que Astro?',
 	'concepts/islands': 'Ilhas Astro',
 
-	tutorials: 'Tutoriais',
-	'blog-tutorial': 'Construa um Blog',
-
-	basics: 'Fundamentos',
+	basics: 'Aprenda os Fundamentos',
 	'basics/project-structure': 'Estrutura de Projetos',
 	'basics/astro-components': 'Componentes',
 	'basics/astro-pages': 'Páginas',
 	'basics/layouts': 'Layouts',
 	'basics/astro-syntax': 'Sintaxe do Astro',
 	'basics/rendering-modes': 'Modos de Renderização',
+	'guides/imports': 'Importações',
+	'guides/integrations-guide': 'Integrações',
 
-	'guides/dev-toolbar': 'Barra de Ferramentas de Desenvolvimento',
-
-	examples: 'Receitas',
-	'guides/migrate-to-astro': 'Migre para o Astro',
+	content: 'Gerencie seu Conteúdo',
+	'guides/content': 'Criando Conteúdo',
+	'guides/markdown-content': 'Markdown e MDX',
+	'guides/content-collections': 'Coleções de Conteúdo',
 	'guides/cms': 'Conecte um CMS',
-	'guides/backend': 'Adicione serviços de backend',
-	'guides/integrations-guide': 'Adicione integrações',
-	'guides/deploy': 'Faça deploy do seu site',
-	'guides/recipes': 'Mais Receitas',
+	'guides/rss': 'Adicione um feed RSS',
 
-	features: 'Guias',
-	'guides/framework-components': 'Frameworks de UI',
+	routes: 'Rotas e Navegação',
 	'guides/routing': 'Roteamento',
-	'guides/markdown-content': 'Markdown e MDX',
-	'guides/content-collections': 'Coleções de Conteúdo',
-	'guides/client-side-scripts': 'Scripts & Manipulação de Eventos',
+	'guides/endpoints': 'Endpoints',
+	'guides/prefetch': 'Pré-carregamento',
+	'guides/middleware': 'Middleware',
+	'guides/internationalization': 'Internacionalização',
+	'guides/view-transitions': 'Transições entre Páginas',
```

Comment:
We are using a different translation on the page itself. Better keep them the same.
```suggestion
	'guides/view-transitions': 'Transições de Visualização',
```

Reply:
Wonderful. Fixed on https://github.com/withastro/docs/pull/8936/commits/c57e459becb1a10ec9c18d3f8b4dc2a75652cecf.